### PR TITLE
Update Chromium/Firefox versions for Navigator API

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects the real values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) and Firefox (Desktop/Android) for the `Navigator` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v4.0.0).

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/Navigator

_Check out the [collector's guide on how to review this PR](https://github.com/foolip/mdn-bcd-collector#reviewing-bcd-changes)._

### CODE REVIEW

File: api/Navigator.json [hunk 1]
```diff
@@ -3411,15 +3411,15 @@
           "spec_url": "https://w3c.github.io/badging/#setappbadge-method",
           "support": {
             "chrome": {
-              "version_added": "83",
+              "version_added": "81",
```
Comment (foolip) [inline api/Navigator.json:3414]: Should the `clearAppBadge` data also be updated, or weren't the shipped at the same time?

File: api/Navigator.json [hunk 2]
```diff
@@ -1306,7 +1306,7 @@
               "version_added": "29"
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "32"
```
Comment (foolip) [inline api/Navigator.json:1309]: Is this based on testing Firefox for Android 29-32? Or why not 29 here?

File: api/Navigator.json [hunk 3]
```diff
@@ -3411,15 +3411,15 @@
...
           "support": {
             "chrome": {
-              "version_added": "83",
+              "version_added": "81",
               "partial_implementation": true,
               "notes": "Windows and macOS only."
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "81"
```
Comment (foolip) [inline api/Navigator.json:3419]: Is this based on testing Chrome for Android? It wouldn't be surprising if this API was shipped at a different time on desktop platforms and Android.

File: api/Navigator.json [hunk 4]
```diff
@@ -4303,7 +4303,7 @@
               "version_added": "8.0"
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "63"
```
Comment (foolip) [inline api/Navigator.json:4306]: Although I don't think WebView can be controlled by WebDriver, the `navigator.webdriver` property should exist and return false, so this seems right.

File: api/Navigator.json (hunk 3 above)
Comment (queengooborg) [reply api/Navigator.json:3419]: This was, yes -- the rest of the Android browsers were mirrored, but testing those in their latest versions showed support.

File: api/Navigator.json (hunk 2 above)
Comment (queengooborg) [reply api/Navigator.json:1309]: Support for the Gamepad API was added in Firefox Android 32 per our data, so this version number is offset to match that of the Gamepad interface.

**Review by foolip — APPROVED** [review]
Before landing, can you update the description to reflect that Firefox data is also updated?

**Comment by queengooborg** [comment]
Done!

**Review by foolip — APPROVED** [review]

**Pull request merged**